src/ngraph_builder.cc

```diff
@@ -59,8 +59,7 @@ static tf::Status MakeConstOp(tf::Node* op, ng::element::Type et,
   return tf::Status::OK();
 }
 
-template <typename T>
-static tf::Status TranslateUnaryOp(tf::Node* op, Builder::OpMap& ng_op_map) {
+static tf::Status TranslateUnaryOp(tf::Node* op, Builder::OpMap& ng_op_map, std::function<std::shared_ptr<ng::Node>(std::shared_ptr<ng::Node>)> f) {
```

Comment:
`f` is a bad parameter name. Please specify something more descriptive. Also provide an example usage.
Also the line is too long - didn't clang-format fix it?